components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateTimePicker.jsx

```diff
@@ -78,49 +78,125 @@ export default class DateTimePicker extends React.Component {
         } else {
             days = 30;
         }
-
-        for (let i = 1; i <= days; i++) {
-            dayComponents.push(
-                <MenuItem
-                    key={`$days-${i}`}
-                    value={i}
-                    primaryText={i}
-                />
-            );
+        if(inputName === 'startTime') {
```

Comment:
formatting issues. No space after if

Reply:
This code is refactored with PR https://github.com/wso2/analytics-solutions/pull/270